packages/gatsby-theme-bulmaio/src/components/navbar/NavbarBrand.tsx

```diff
@@ -20,21 +20,9 @@ class NavbarBrand extends React.Component<NavbarBrandProps> {
   };
 
   public render() {
-    const { githubUrl, twitterUrl } = this.props;
     return (
       <div className="navbar-brand">
         <NavbarBrandLogo/>
-        <a data-testid={`navbarbrand-github`} className="navbar-item is-hidden-desktop bio-navbar-brand-github" href={githubUrl} target="_blank">
```

Comment:
Any reason for dropping this?

Reply:
Pycharm had duplicate Twitter and GitHub icons at the beginning and the end of the Navbar. It was weird looking. I also don't see that behavior in production, so maybe it was an accident to add them in the first place.